src/github_runner_image_builder/cli.py

```diff
@@ -350,3 +361,16 @@ def run(  # pylint: disable=too-many-arguments, too-many-locals, too-many-positi
     if callback_script:
         # The callback script is a user trusted script.
         subprocess.check_call([str(callback_script), image_ids])  # nosec: B603
+
+
+def _load_secrets() -> dict[str, str]:
+    """Load image builder secrets set as environment variables.
+
+    Returns:
+        The secrets key value pairs.
+    """
+    secrets: dict[str, str] = {}
+    for env in os.environ:
+        if env.startswith(SECRET_PREFIX):
+            secrets[env.removeprefix(SECRET_PREFIX)] = os.environ[env]
+    return secrets
```

Comment:
I think you can do this with dictionary comprehension:

return {key: value for key, value in os.environ if key.startswith(SECRET_PREFIX)}